src/30_Advanced/SeatMap.html

```diff
@@ -0,0 +1,175 @@
+    `amp-pan-zoom` for making the seatmap zoomable and scrollable -->
+    <script async custom-element="amp-pan-zoom" src="https://cdn.ampproject.org/v0/amp-pan-zoom-0.1.js"></script>
+    <!-- `amp-bind` for enabling seat selection -->
+    <script async custom-element="amp-bind" src="https://cdn.ampproject.org/v0/amp-bind-0.1.js"></script>
+    <!-- `amp-list` for showing a list of seats -->
+    <script async custom-element="amp-list" src="https://cdn.ampproject.org/v0/amp-list-0.1.js"></script>
+    <!-- `amp-mustache` for rendering the `amp-list` content -->
+    <script async custom-template="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.2.js"></script>
+    <style amp-custom>
+      :root {
+      --abe-light-pink: rgba(182, 8, 69, 0.5);
+      --abe-pink: rgb(182, 8, 69);
+      }
+      .seat {
+        stroke-width: 3;
+        stroke: rgb(0,0,0);
+        fill:white;
+      }
+      .unavailable {
+        fill: var(--abe-light-pink);
+        pointer-events: none;
+      }
+      .selected {
+        fill: var(--abe-pink);
+      }
+      .disabled {
+        pointer-events: none;
+        cursor: default;
+      }
+      .zoom-button {
+        bottom: 60px;
+        right: 0px;
+        z-index: 1;
+        border-color: Transparent;
+        background-color: Transparent;
+        margin: 8px;
+        width: 40px;
+        height: 40px;
+        padding: 0;
+        border-width: 0;
+        box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.25);
+        background-color: white;
+        border-radius: 3px;
+      }
+      .opaque {
+        opacity: 0.5;
+        filter: alpha(opacity=50);
+      }
+      .seat-map-svg {
+        box-sizing: border-box;
+        padding-top: 30px;
+      }
+      #pan-zoom {
+        display: initial;
+      }
+      #pan-zoom > svg {
+        left: 0;
+        right: 0;
+        margin: auto;
+      }
+      .list-container {
+        position: relative;
+        height: 50vh;
+      }
+      .ampstart-card {
+        min-width: 100%;
+        padding: 1rem;
+      }
+    </style>
+    <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style>
+    <noscript>
+      <style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style>
+    </noscript>
+
+  </head>
+  <body>
+    <!-- ## Seatmap -->
+    <!-- Here we are using an `amp-state` called `seatmap`.
```

Comment:
Please rephrase the first sentence, give a more high level intro into what you're doing.